testing/asserts.ts

```diff
@@ -12,9 +12,12 @@ import {
   stripColor,
   white,
 } from "../fmt/colors.ts";
+import { fromFileUrl, join, parse } from "../path/mod.ts";
+import { ensureFile, ensureFileSync } from "../fs/mod.ts";
```

Comment:
The tests are failing, because this module is marked as "browser compatible" - see comment at the top of the file. Maybe let's move it to `testing/snapshot.ts` for now, since snapshotting requires to write files to disk and can't be done in browsers.

Reply:
@bartlomieju this is done now 🙂